src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -17,12 +17,8 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "symfony/config": "~3.4|~4.0",
-        "symfony/twig-bridge": "^3.4.3|^4.0.3",
-        "symfony/http-foundation": "~3.4|~4.0",
-        "symfony/http-kernel": "~3.4|~4.0",
-        "symfony/polyfill-ctype": "~1.8",
-        "twig/twig": "~1.34|~2.4"
+        "symfony/twig-bundle": "^3.4.3|^4.0.3",
```

Comment:
You're looking for troubles by requiring it itself 😅
Should actually be in `Symfony/Bundle/WebProfilerBundle/composer.json`, not TwigBundle's composer.json.